lib/trln_argon/controller_override/local_filter.rb

```diff
@@ -5,6 +5,8 @@ module LocalFilter
 
       included do
         helper_method :filtered_results_total
+        helper_method :filters_applied?
```

Comment:
I don't think these methods belong here. This controller module is concerned with displaying the result counts on the Local/Trln toggle filter. I think these methods would be more appropriate in `cd lib/trln_argon/view_helpers` perhaps in a new module just for zero results helpers, maybe `ZeroResultsHelper`.